pytest_tap/tests/test_plugin.py

```diff
@@ -72,15 +72,15 @@ def test_tracks_ok(self):
         report = mock.Mock(when='call', outcome='passed', location=location)
         plugin.pytest_runtest_logreport(report)
         plugin.tracker.add_ok.assert_called_once_with(
-            'test_file.py', 'TestFake.test_me')
+                'test_file.py', 'test_file.py::TestFake.test_me')
```

Comment:
Looks like there are extra spaces here. Could you update this so the indentation is only 4 spaces?